packages/contentful/package.json

```diff
@@ -17,6 +17,12 @@
     "schema:update:crn": "graphql-codegen --config codegen.yml"
   },
   "dependencies": {
+    "@asap-hub/errors": "workspace:*",
+    "@asap-hub/model": "workspace:*",
+    "@babel/runtime-corejs3": "7.19.6",
+    "@contentful/rich-text-html-renderer": "^15.13.1",
+    "@contentful/rich-text-types": "^15.13.2",
+    "contentful": "beta-v10",
```

Comment:
This was installed in this specific beta version to make it possible to use typescript https://www.contentful.com/developers/docs/javascript/tutorials/typescript-in-javascript-client-library/